maro/simulator/core.py

```diff
@@ -126,8 +129,7 @@ def frame_index(self) -> int:
 
     @property
     def snapshot_list(self) -> SnapshotList:
-        """SnapshotList: A snapshot list contains all the snapshots of frame at each tick.
-        """
+        """SnapshotList: A snapshot list containing all the snapshots of frame at each tick."""
```

Comment:
at each dump point (the resolution of the snapshot may be different from the simulation frame resolution.)